src/components/document/student-group-view.tsx

```diff
@@ -43,13 +43,24 @@ export const StudentGroupView:React.FC<IProps> = ({ groupId, setGroupId }) => {
   const [focusedGroupUser, setFocusedGroupUser] = useState<GroupUserModelType | undefined>();
   const selectedGroupId = groupId || (groups.allGroups.length ? groups.allGroups[0].id : "");
   const groupUsers = getGroupUsers(user.id, groups, documents, selectedGroupId);
-  const group = groups.getGroupById(selectedGroupId);
-
   const ui = useUIStore();
+  const group = groups.getGroupById(selectedGroupId);
   const isStudentViewActiveTab = (ui.activeNavTab === "student-work");
   const isChatPanelShown = ui.showChatPanel;
   const shrinkStudentView  = isStudentViewActiveTab && isChatPanelShown;
-  const classes = classNames("document", "student-group-view", {"shrink-student-view": shrinkStudentView});
+  const documentSelectedForComment = ui.showChatPanel && ui.selectedTileIds.length === 0;
```

Comment:
Have to add whether there is a focused document to this logic. Otherwise, the 4up documents are highlighted when you open the chat panel.